API/DTOs/TournamentPlatformStatsDTO.cs

```diff
@@ -16,4 +17,19 @@ public class TournamentPlatformStatsDTO
     /// Map of <see cref="Common.Enums.Verification.VerificationStatus"/>es to the number of <see cref="Database.Entities.Tournament"/>s with the status
     /// </summary>
     public Dictionary<VerificationStatus, int> CountsByVerificationStatuses { get; init; } = new();
```

Comment:
Would rename this to `CountByVerificationStatus` as well.

Reply:
I thought for a while on this, I think CountBy works best because:
- It sounds similar to other EF query extension methods
- The name more or less supports the idea that the result is grouped.

The "technically correct" name would be something like GroupByCountByVerificationStatus, but that's a mouthful. We can let the documentation do the heavy lifting for us.